src/plugins/data-card/components/sort-stack.tsx

```diff
@@ -40,13 +92,56 @@ export const SortStack: React.FC<IProps> = ({ model, stackValue, inAttributeId,
    { "is-over" : isOver}
   );
 
+  const cellStackClasses = classNames("cell stack",
```

Comment:
This looks like one class but is, I guess, actually two.  Probably better to list as `"cell", "stack"` or use `"cell-stack"`.